core/src/com/unciv/logic/trade/TradeOffer.kt

```diff
@@ -38,7 +38,11 @@ data class TradeOffer(val name: String, val type: TradeType, var amount: Int = 1
         var offerText = when(type){
             TradeType.WarDeclaration -> "Declare war on [$name]"
             TradeType.Introduction -> "Introduction to [$name]"
-            TradeType.City -> UncivGame.Current.gameInfo!!.getCities().firstOrNull{ it.id == name }?.name ?: "Non-existent city"
+            TradeType.City -> {
+                val city =
+                        UncivGame.Current.gameInfo!!.getCities().firstOrNull { it.id == name }
+                if (city != null) city.name + " (" + city.population.population + ")" else "Non-existent city"
```

Comment:
You need to either.tr() city name or wrap in curly brackets because it's not going to be translated like this

Reply:
Done